test/integration/RpParsingTest.php

```diff
@@ -47,6 +47,20 @@ public function testParseRpCustom(){
         $item = $qtiParser->load();
         $this->assertInstanceOf('\\oat\\taoQtiItem\\model\\qti\\Item',$item);
         $this->assertInstanceOf('\\oat\\taoQtiItem\\model\\qti\\response\\Custom',$item->getResponseProcessing());
+
+        /**
+         * orphaned response conditions must lead to a custom response processing
```

Comment:
This modification has been validated by Sam, it's a changed behaviour in the library.